templates/components/radio.build.hbs

```diff
@@ -1,24 +1,9 @@
 <template>
   <div v-if="description" class="help-block description">\{{ description }}</div>
-  <template v-if="readonly">
-    <template v-for="(option, index) in options">
-      <template v-if="value === option.label || value === option.id">
-        <p class="form-control-static">\{{ option.label || option.id }}</p>
-        <input
-          type="hidden"
-          :id="name + '-' + index"
-          :name="name"
-          v-model="value"
-          :value="option.id || option.label"
-          class="focus-outline"
-          tabindex="0"
-        >
-      </template>
-    </template>
-  </template>
-  <template v-else>
+
```

Comment:
Can you please remove new empty blank line

Reply:
done changes